Tetrahedral_remeshing/include/CGAL/Tetrahedral_remeshing/internal/tetrahedral_remeshing_helpers.h

```diff
@@ -761,20 +763,20 @@ Subdomain_relation compare_subdomains(const typename C3t3::Vertex_handle v0,
 {
   typedef typename C3t3::Subdomain_index Subdomain_index;
 
-  std::vector<Subdomain_index> subdomains_v0;
-  incident_subdomains(v0, c3t3, std::back_inserter(subdomains_v0));
-  std::sort(subdomains_v0.begin(), subdomains_v0.end());
+  boost::container::flat_set<Subdomain_index> subdomains_v0;
```

Comment:
Use a `small_vector` as the third template arguments of the `flat_set`. That will avoid the memory allocation at each call of the function.